api/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>com.lishid</groupId>
     <artifactId>openinvparent</artifactId>
-    <version>4.0.9-SNAPSHOT</version>
+    <version>4.0.10-SNAPSHOT</version>
```

Comment:
Usually you don't want to touch the version in PRs, as the repo
- may have other changes to include in a release
- will cause merge conflicts with future PRs (until pulled)

Reply:
Also, this would make the PR a much more manageable one-line change instead of a larger 10 file change.